portfolio/src/main/webapp/skills.html

```diff
@@ -13,91 +13,83 @@
   <body>
     <nav class="navbar navbar-expand-md navbar-dark bg-primary mb-3">
       <div class="container-fluid">
-          <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#navbarCollapse">
-              <span class="navbar-toggler-icon"></span>
-          </button>
-          <div class="collapse navbar-collapse" id="navbarCollapse">
-              <div class="navbar-nav">
-                  <a class="nav-item nav-link" href="index.html"> <span class="link"> Home </span> </a>
-                  <a class="nav-item nav-link" href="about.html"> <span class="link"> About </span> </a>
-                  <a class="nav-item nav-link" href="skills.html"> <span class="link"> Skills </span> </a>
-                  <a class="nav-item nav-link" href="projects.html"> <span class="link"> Projects </span> </a>
-              </div>
+        <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#navbarCollapse">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarCollapse">
+          <div class="navbar-nav">
+            <a class="nav-item nav-link" href="index.html"> <span class="link"> Home </span> </a>
+            <a class="nav-item nav-link" href="about.html"> <span class="link"> About </span> </a>
+            <a class="nav-item nav-link" href="skills.html"> <span class="link"> Skills </span> </a>
+            <a class="nav-item nav-link" href="projects.html"> <span class="link"> Projects </span> </a>
           </div>
+        </div>
       </div>    
     </nav>
     <div class="text-center container"> 
-      <div class="skill-title"> Skills </div>
-      <div class="description"> I love learning and building! </div>
-      <div class="description"> Here is some of what I've picked up along the way. </div>
-
+      <div class="title"> Skills </div>
+      <div class="tag-line"> I love learning and building! </div>
+      <div class="tag-line"> Here is some of what I've picked up along the way. </div>
       <div>
         <div class="skill-category"> General Languages </div>
-            <div class="skill-list">
-                <span className="skill-item"> Java </span>
-                <span className="skill-item"> Python </span>
-                <span className="skill-item"> C </span>
-                <span className="skill-item"> C++ </span>
-                <span className="skill-item"> Javascript </span>
-            </div>
+          <div class="skill-list">
```

Comment:
These should be list elements (`<ul>` or `<ol>` depending on whether order matters), and the `<span>`s should be `<li>`s. This is especially helpful for navigating the page using a screen reader. If you don't want the default styling that comes with these elements (like the bullets, for example) you can alter them with CSS.

Reply:
got it. thanks!